test.ts

```diff
@@ -96,6 +97,20 @@ test('is500', t => {
 	t.false(pageDetect.is500());
 });
 
+test('isPRCommit404', t => {
+	document.title = 'Commit range not found · Pull Request #3227 · sindresorhus/refined-github';
+	jsdom.reconfigure({url: 'https://github.com/sindresorhus/refined-github/pull/3227/commits/32c8a88360a85739f151566eae0225d530ce6a15'});
```

Comment:
Thanks for adding tests for this!

You can actually just use `location.href = 'https://github.com/sindresorhus/refined-github/pull/3227/commits/32c8a88360a85739f151566eae0225d530ce6a15'` instead of changing the JSDom export

Reply:
I tried that first, but it yields the following error:

```
Error: Not implemented: navigation (except hash changes)
    at module.exports (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/browser/not-implemented.js:9:17)
    at navigateFetch (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/window/navigation.js:76:3)
    at exports.navigate (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/window/navigation.js:54:3)
    at LocationImpl._locationObjectNavigate (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/window/Location-impl.js:31:5)
    at LocationImpl._locationObjectSetterNavigate (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/window/Location-impl.js:25:17)
    at LocationImpl.set href [as href] (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/window/Location-impl.js:47:10)
    at Location.set href [as href] (/home/flo/www/github-url-detection/node_modules/jsdom/lib/jsdom/living/generated/Location.js:116:37)
    at /home/flo/www/github-url-detection/test.ts:102:15
    at Test.callFn (/home/flo/www/github-url-detection/node_modules/ava/lib/test.js:596:21)
    at Test.run (/home/flo/www/github-url-detection/node_modules/ava/lib/test.js:609:23)
```

That's why I went with the JSDom approach, which needs these few minor changes at the top of the file.

Reply:
We used to do exactly that before, perhaps JSDom stopped supporting it:

https://github.com/sindresorhus/refined-github/blob/90b43ae76d84b53eb01a1d8961907c515c010994/test/page-detect.ts#L35

Reply:
Very strange, but at least the results on GitHub Actions' runner are consistent with mine:
https://github.com/fregante/github-url-detection/pull/12/checks?check_run_id=785944721

Maybe one of the other updated packages is to blame?

Reply:
I see why. We used to have this line:

https://github.com/fregante/github-url-detection/blob/99f0762f571d2a494c6a624729624f96c92725aa/test.ts#L10

Now we have

https://github.com/FloEdelmann/github-url-detection/blob/d724f4779547e7e82fbad133c9882d833a3aa257/test.ts#L10